lib/src/emoji_picker.dart

```diff
@@ -181,6 +181,8 @@ class _EmojiPickerState extends State<EmojiPicker> {
           await _getAvailableEmojis(emoji_list.animals, title: 'animals')),
       CategoryEmoji(Category.FOODS,
           await _getAvailableEmojis(emoji_list.foods, title: 'foods')),
+          CategoryEmoji(Category.ACTIVITIES,
+          await _getAvailableEmojis(emoji_list.activities, title: 'activities')),
```

Comment:
@AndriiChuzhynov Can you fix the formatting?

Reply:
Sure, just fixed. 
I've also touched `emoji_picker_flutter.dart` because `flutter analyze lib test example` returned 1 with the message:
`info • Sort directive sections alphabetically • lib/emoji_picker_flutter.dart:5:1 • directives_ordering`